plugin/hpsa/hpsa.py

```diff
@@ -755,7 +761,7 @@ def volume_raid_create(self, name, raid_type, disks, strip_size,
             cmds.append("ss=%d" % int(strip_size / 1024))
 
         try:
-            self._sacli_exec(cmds, flag_convert=False)
```

Comment:
I still really, really don't like this change as default behavior. You should not need to force create a volume. 

From your email a few weeks ago you made it seem like this issue only happened when you tried to change the initial volume setting. Did you encounter behavior where a force create is required at initial deployment?